frontend/src/queries/Query/Query.tsx

```diff
@@ -106,6 +103,8 @@ export function Query(props: QueryProps): JSX.Element | null {
         component = <TimeToSeeData query={query} cachedResults={props.cachedResults} />
     } else if (isWebOverviewQuery(query)) {
         component = <WebOverview query={query} cachedResults={props.cachedResults} context={queryContext} />
+    } else {
```

Comment:
Why not keep the `if (isDataNode(query))`? Seemed safer

Reply:
Essentially everything else that doesn't have a custom visualization should be a data node. We're just not very good at keeping the `isDataNode` function up to date.